### ISSUE TITLE
Doesn't work for Arabic locale

### ISSUE THREAD

**Issue by saeed74** [danikula/AndroidVideoCache#87]
if user has been selected persian or arabic language from settings of the device, the video cache will not work because of arabic/persian digits in pingUrl:

Logcat:
```
Open connection  to http://127.0.0.1:۴۲۵۸۹/ping
Error reading ping response
com.danikula.videocache.ProxyCacheException: Error opening connection for http://127.0.0.1:۴۲۵۸۹/ping with offset 0
```

i just solved the problem by adding a method in HttpProxyCacheServer.java:

Here is Codes:
```
String pingUrl = appendToProxyUrl(PING_REQUEST);
String CorrectpingUrl = convertToEnglishDigits(pingUrl);
HttpUrlSource source = new HttpUrlSource(CorrectpingUrl);
```

and here is the convertToEnglishDigits method code:

```
public static String convertToEnglishDigits(String value)
{
    String newValue = value.replace("١", "1").replace("٢", "2").replace("٣", "3").replace("٤", "4").replace("٥", "5")
            .replace("٦", "6").replace( "٧","7").replace("٨", "8").replace("٩", "9").replace("٠", "0")
            .replace("۱", "1").replace("۲", "2").replace("۳", "3").replace("۴", "4").replace("۵", "5")
            .replace("۶", "6").replace("۷", "7").replace("۸", "8").replace("۹", "9").replace("۰", "0");
    return newValue;
}
```

**Comment by saeed74** [comment]
Oh And Another Problem When Language is Persian/Arabic:
When Useing this code:

```
HttpProxyCacheServer proxy = App.getProxy(context);
String proxyUrl = proxy.getProxyUrl(data);
mPlayer.setDataSource(ProxyUrl);
```

the proxyUrl will be like below:

`http://127.0.0.1:۵۷۹۷۱/https%3A%2F%2Fdl.dropboxusercontent.com%2Fs%2Fb0pbv3c1y6ssw16%2Fmardome_shahr.mp3`

that is obvious will not work and it respond with MediaPlayer Error (1, -1004).
I used convertToEnglishDigits method again for handle this:

```
HttpProxyCacheServer proxy = App.getProxy(context);
String proxyUrl = proxy.getProxyUrl(data);
String correctProxyUrl = App.convertToEnglishDigits(proxyUrl);
mPlayer.setDataSource(correctProxyUrl);
```

btw Thank You for Your Awesome Library<3

**Comment by danikula** [comment]
@saeed74 fixed in 2.6.1. Thanks for your contribution.
